deps/graph-parser/src/logseq/graph_parser/block.cljs

```diff
@@ -464,7 +475,7 @@
 
 (defn get-page-refs-from-properties
   [format properties db date-formatter]
-  (let [page-refs (get-page-ref-names-from-properties format properties)]
+  (let [page-refs (get-page-ref-names-from-properties format properties {})]
```

Comment:
I didn't pass the user config here as I didn't understand if this config mattered here. If it does, there is some small work to find all callers to this fn outside the graph-parser and pass in `(state/get-config)` to it